ui/uihandler.go

```diff
@@ -40,7 +42,7 @@ var (
 	// UI and Debug prefix/paths (read in ui handler).
 	uiPath      string // absolute (base)
 	logoPath    string // relative
-	chartJSPath string // relative
+  chartJSPath string // relative
```

Comment:
make sure to run `go fmt` at save time with your editor (or run it manually)

Reply:
turns out go fmt doesn't automatically turn spaces to tabs. Done manually.